.gitignore

```diff
@@ -1,4 +1,6 @@
 /web/bundles/
+/web/js/
+/web/css/
```

Comment:
this is where [CreateBundle dumps its asset files](https://github.com/symfony-cmf/CreateBundle/blob/master/Resources/views/includejsfiles-create.html.twig#L24). which sounds like a less-than-ideal choice as this risks name clashes.

one way to fix this without major BC issues could be to change the path to web/js/create/ and web/css/create and change the ignore to only the subfolder. wdyt?